src/v2/styles/MessageReactions/MessageReactions-layout.scss

```diff
@@ -104,12 +104,19 @@
       padding: var(--str-chat__spacing-1) 0;
       flex-shrink: 0;
       cursor: pointer;
+
+      .str-chat__message-reaction-emoji {
+        width: 18px;
```

Comment:
Why do we need to use a `px` value here? Can't we use `rem`?

Reply:
So this is added for compatibility with `StreamEmoji` which currently has size hard-coded at `18px` (see https://github.com/GetStream/stream-chat-react/blob/master/src/components/Reactions/StreamEmoji.tsx#L29) - so we have to use the same size here, unfortunately.

Reply:
I see, it's an image. Let me check if this works with Angular

Reply:
Based on my tests, I don't think we're intersecting here - let me know if there are some conflicts I missed.

Reply:
I just did a test, and I'm seeing what's on the screenshot above. This is what I'm seeing in the inspector:

![Screenshot 2024-02-13 at 16 01 03](https://github.com/GetStream/stream-chat-css/assets/6690098/b2af694b-53a6-4a5f-8f2d-f55280bb3902)

Reply:
My bad, I probably didn't update styles in Angular properly when testing. Let me fix that :)

Reply:
No problem, thanks :)

Reply:
Fixed in 4eeb2116acb4f606a375bc05e3b3867de5fe2001.

Reply:
Thank you